src/org/jitsi/impl/neomedia/rtp/RTPEncodingDesc.java

```diff
@@ -761,6 +755,20 @@ FrameDesc getLastReceivedFrame()
      */
     public RTPEncodingDesc getBaseLayer()
     {
+        if (!baseIsInitialized)
+        {
+            if (ArrayUtils.isNullOrEmpty(dependencyEncodings))
+            {
+                base = this;
+            }
+            else
+            {
+                base = dependencyEncodings[0].getBaseLayer();
```

Comment:
Just a general concern: signaling a circular dependency will result in an infinite loop here. At some layer between the client and this code (maybe jicofo, maybe jvb) we need to ensure this doesn't happen.

Reply:
Looks like the bridge ensure this.